openshift/generate-manifests.sh

```diff
@@ -54,7 +54,8 @@ mkdir -p "${TMP_ROOT}/openshift"
 cp -a "${REPO_ROOT}/openshift/kustomize" "${TMP_ROOT}/openshift/kustomize"
 
 # Override OPENSHIFT-NAMESPACE to ${NAMESPACE}
-find "${TMP_ROOT}" -name "*.yaml" -exec sed -i "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
+find "${TMP_ROOT}" -name "*.yaml" -exec sed -i'.bak' "s/OPENSHIFT-NAMESPACE/${NAMESPACE}/g" {} \;
+find "${TMP_ROOT}" -name "*.bak" -exec rm {} \;
```

Comment:
actually... no exec is needed, the action could just be `-delete`:
```
find "${TMP_ROOT}" -name "*.bak" -delete

Reply:
https://github.com/openshift/operator-framework-operator-controller/pull/232#discussion_r1925491963

If that passes in the CI, we remove it in the follow-up.
I have the commit with the catalogd already, but I cannot push another PR with the same commits, etc, required so if you agree, I think that is the easy away.

Reply:
Not blocking... just an optimization